.github/ISSUE_TEMPLATE/documentation.yml

```
@@ -0,0 +1,47 @@
+name: Documentation
+description: Report problems and suggest ideas for the documentation
```

Minor and optional, only update this if you agree :slightly_smiling_face: 

```suggestion
description: Report problems with or suggest ideas for the documentation
```

---

I'm not sure, "Report problems for the documentation" makes sense to me.